vue/test/components/scrollable-items/scrollable-items.test.js

```diff
@@ -0,0 +1,394 @@
+const assert = require("assert");

```

Comment:
Can you enhance this with scroll position assertions? I'd argue that it is the part of the implementation most sensitive to errors.

Reply:
Nice work with the tests btw

Reply:
Can't do, because of the problems [I commented here](https://github.com/ripe-tech/ripe-components-vue/pull/418#discussion_r553898845).